docs/ops/sort/MatrixNMS_8.md

```diff
@@ -0,0 +1,157 @@
+3.  Take th column-wise max of `X` to compute a vector `K` of maximun IoU for each candidate box.
+4.  Repeat element value of `K` along axis 1. Suppose this get the matrix `X_cmax`.
+5.  Compute the decay factor: `decay_factor = exp((X_cmax**2 - X**2) * gaussian_sigma)` if `decay_function` is `guassian`, else `decay_factor = (1 - X) / (1 - X_cmax)`.
+6.  Take the column-wise min of `decay_factor`, and element-wise multiply with scores to decay them.
+
+**Attributes**:
+
+* *sort_result*
+
+  * **Description**: *sort_result* specifiy order of output elements.
+  * **Range of values**: `class`, `score`, `none`
+    * *class* - sort selected boxes by class id (ascending) in each batch element.
+    * *score* - sort selected boxes by score (descending) in each batch element.
+    * *none* - do not guarantee the order in each batch element.
+  * **Type**: `string`
+  * **Default value**: `none`
+  * **Required**: *No*
+
+* *output_type*
+
+  * **Description**: the output tensor type
+  * **Range of values**: `i64` or `i32`
+  * **Type**: `string`
+  * **Default value**: `i32`
+  * **Required**: *No*
+
+* *score_threshold*
+
+  * **Description**: minimum score to consider box for the processing.
+  * **Range of values**: a floating-point number
+  * **Type**: `float`
+  * **Default value**: `0`
+  * **Required**: *No*
+
+* *nms_top_k*
+
+  * **Description**: maximum number of boxes to be selected per class.
+  * **Range of values**: an integer starting from `-1`.
+  * **Type**: `int`
+  * **Default value**: `-1` meaning to keep all boxes
+  * **Required**: *No*
+
+* *keep_top_k*
+
+  * **Description**: maximum number of boxes to be selected per batch element.
+  * **Range of values**: an integer starting from `-1`.
+  * **Type**: `int`
+  * **Default value**: `-1` meaning to keep all boxes
+  * **Required**: *No*
+
+* *background_class*
+
+  * **Description**: the background class id.
+  * **Range of values**: an integer from `-1` to the maximum class id
+  * **Type**: `int`
+  * **Default value**: `-1` meaning to keep all classes
+  * **Required**: *No*
+
+* *decay_function*
+
+  * **Description**: decay function used to decay scores.
+  * **Range of values**: `gaussian`, `linear`
+  * **Type**: `string`
+  * **Default value**: `linear`
+  * **Required**: *No*
+
+* *gaussian_sigma*
+
+  * **Description**: gaussian_sigma parameter for gaussian decay_function.
+  * **Range of values**: a floating-point number
+  * **Type**: `float`
+  * **Default value**: `2.0`
+  * **Required**: *No*
+
+* *post_threshold*
+
+  * **Description**: threshold to filter out boxes with low confidence score `after` decaying.
+  * **Range of values**: a floating-point number
+  * **Type**: `float`
+  * **Default value**: `0`
+  * **Required**: *No*
+
+**Inputs**:
+
+*   **1**: `boxes` - tensor of type *T* and shape `[num_batches, num_boxes, 4]` with box coordinates. he box cooridnates are layout as `[xmin, ymin, xmax, ymax]`. Required.
```

Comment:
Wrap word "Required" with double "*" characters.

Reply:
Is there any unified specification about the format? I checked, most Op has no decoration to "Required". Only a small part of Ops decorated to "Required". Single "*" wrap. @lazarevevgeny